CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to vanilla-commons
+First off, thanks for taking the time to contribute!
+
+## Publish
+
+Every pull request will release a new version of xcribe on hex.pm.
+So make sure to update the changelog and bump the version before merging the PR.
+
+Run `make release` and follow the instructions. After that, update `CHANGELOG.md`
+with a description of what is been changed in the release.
```

Comment:
So now the release flow is to run `make release` within the dev branch; then, after merged, push the new version tag? Is that correct?

Reply:
Yes @brosquinha . A next step would be to add the `push the new version tag` to the CI flow, then everything would be automated.